privaterelay/settings.py

```diff
@@ -867,6 +867,12 @@ def _get_initial_middleware() -> list[str]:
     }
     SESSION_ENGINE = "django.contrib.sessions.backends.cache"
     SESSION_CACHE_ALIAS = "default"
+if RELAY_CHANNEL == "local":
+    CACHES = {
+        "default": {
+            "BACKEND": "django.core.cache.backends.locmem.LocMemCache",
+        }
+    }
```

Comment:
(question) Are we using this instead of Redis on local? If so this is better set as `elif`